### ISSUE TITLE
[Dark Sky Migration] Separate sensor entities error

### ISSUE THREAD

**Issue by qJake** [alexander0042/pirate-weather-ha#92]
I'm working on replacing the old Dark Sky entity (for me, it was `weather.dark_sky`) with Pirate Weather.

In my configuration, I have selected both "Weather" and "Sensor" entities.

The Sensor platform is not loading correctly for me:

![image](https://user-images.githubusercontent.com/652006/228325185-d5aba8c4-53b2-4dc1-a0f1-152748de7268.png)

Here's my configuration:

![image](https://user-images.githubusercontent.com/652006/228325657-64699d94-0c22-4e62-8816-93c85be9c731.png)

Here is the HA error log details:

```
Logger: homeassistant.loader
Source: custom_components/pirateweather/sensor.py:217
Integration: Pirate Weather ([documentation](https://github.com/alexander0042/pirate-weather-ha), [issues](https://github.com/alexander0042/pirate-weather-ha/issues))
First occurred: 12:40:31 PM (1 occurrences)
Last logged: 12:40:31 PM

Unexpected exception importing platform custom_components.pirateweather.sensor
Traceback (most recent call last):
  File "/usr/src/homeassistant/homeassistant/loader.py", line 728, in get_platform
    cache[full_name] = self._import_platform(platform_name)
  File "/usr/src/homeassistant/homeassistant/loader.py", line 745, in _import_platform
    return importlib.import_module(f"{self.pkg_path}.{platform_name}")
  File "/usr/local/lib/python3.10/importlib/__init__.py", line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1050, in _gcd_import
  File "<frozen importlib._bootstrap>", line 1027, in _find_and_load
  File "<frozen importlib._bootstrap>", line 1006, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 688, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 883, in exec_module
  File "<frozen importlib._bootstrap>", line 241, in _call_with_frames_removed
  File "/config/custom_components/pirateweather/sensor.py", line 217, in <module>
    device_class=SensorDeviceClass.PRECIPITATION,
  File "/usr/local/lib/python3.10/enum.py", line 437, in __getattr__
    raise AttributeError(name) from None
AttributeError: PRECIPITATION
```

Any ideas what I'm doing wrong?

**Comment by 26tajeen** [comment]
[Ah! Exact same issue here](https://community.home-assistant.io/t/is-dark-sky-really-finally-dead/551072/9?u=daneboom)!

**Comment by qJake** [comment]
I fixed it! 😁 I was on an old version of HA (2022.11.x) so upgrading to 2023.3.x fixed this for me.

If people are still having this issue while on 2023.3.x I would suggest opening your own issue!
